### ISSUE TITLE
Preview for tables in markdown files is incorrect

### ISSUE THREAD

**Issue by nehashri** [Microsoft/vscode#33369]
- VSCode Version: Code 1.15.1 (41abd21afdf7424c89319ee7cb0445cc6f376959, 2017-08-16T17:15:57.756Z)
- OS Version: Darwin x64 16.7.0
- Extensions: 

Extension|Author (truncated)|Version
---|---|---
vscode-docker|Pet|0.0.16
vscode-eslint|dba|1.2.11
tslint|eg2|0.17.0
Go|luk|0.6.63
debugger-for-chrome|msj|3.2.0
java|red|0.8.0

---
Markdown preview broken for tables.
Steps to Reproduce:
1. Create a markdown file with following contents. 
   Note: there are 3 spaces before start of table.
```
Heading
=======
   |Word |Vowel Count|
   |------|----------|
   |Rhythm|0         |
   |Mingle|2         |
```
2. The preview for the file does not show the first column in the table. 

<img width="740" alt="screen shot 2017-08-29 at 15 37 52" src="https://user-images.githubusercontent.com/5715387/29815881-d9788290-8ccf-11e7-8f5a-1b3b4174bacf.png">

If there are no spaces included before the start of table pipe, the preview of table is rendered correctly. 
Whereas if there are spaces, the table rendering is broken. The contents of the first column is pushed to second column.

Reproduces without extensions: Yes

**Comment by mjbvz** [comment]
We use markdown-it, and the table seems to work fine on the current markdown-it page: https://markdown-it.github.io I suspect we may just need to bump our markdown-it version

**Comment by mjbvz** [comment]
Should be fixed in first 1.17 insiders:

![screen shot 2017-09-01 at 2 40 50 pm](https://user-images.githubusercontent.com/12821956/29988683-9749fe8e-8f23-11e7-8143-b86630a7a2b1.png)
